src/app/components/User/User.tsx

```diff
@@ -12,15 +12,11 @@ const User = (): JSX.Element => {
   return (
     <section className={classes.container}>
       <div className={classes.header}>
-        {currentUser?.avatar ? (
-          <img
-            src={currentUser.avatar}
-            className={classes.avatar}
-            alt="overwolf profile avatar"
-          />
-        ) : (
-          defaultAvatar
-        )}
+        <img
+          src={currentUser?.avatar || defaultAvatar}
+          className={classes.avatar}
+          alt="overwolf profile avatar"
```

Comment:
It's not important for Overwolf apps, but the `alt` could be improved if this would be for a website.
This image is different for the users, thus, the description of the image should be different too.

e.g. `alt="Alexis"` if it is an image of "Alexis".
"Overwolf profile avatar" would be too generic.

Consider setting `alt=""` to remove duplicate information because you already have the information in the adjacent `h1`.

Read this for more information
https://webaim.org/techniques/alttext/

Reply:
okay, definitly something to consider, for this special reason, I would remove it completly. 👍